movai_core_shared/logger.py

```diff
@@ -45,6 +49,18 @@
     datefmt=LOG_FORMATTER_DATETIME,
 )
 
+SEVERETY_CODES = {
```

Comment:
use logging lib standard
like we use envvars
`from logging import DEBUG, NOTSET, INFO`

Reply:
I had to it to be compatible with the syslog format, can you approve that logging format follows syslog protocol specification ? 
[syslog](https://en.wikipedia.org/wiki/Syslog)

Reply:
well after I have checked the codes do not match beterrn syslog and logging so I have to maintain this mapping for all relevant codes.